sass/components/_buttons.scss

```diff
@@ -1,6 +1,15 @@
 @charset 'UTF-8';
 
 button {
+    .unread-badge {
```

Comment:
I think this might cause problems in the desktop app since it uses buttons for its sidebar items instead of links. Is there maybe somewhere more specific we could put this?

Reply:
Never mind. We don't use the `unread-badge` name outside of the team sidebar. The CSS class used elsewhere is `badge`. It would be nice to rename this to avoid any potential naming confusion, but I can't think of anything better to call it at the moment.

Reply:
Thanks for raising this as a potential concern @hmhealey This was tested on desktop as well and so far as I can tell it works the same as webapp.